### DISCUSSION TITLE
Технические работы

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#158] [Объявления]
### Обновление поискового индекса
27.04.2023 23:00 МСК остановлено обновление БД, производятся работы по обновлению поискового индекса.
Последний комментарий в БД до начала тех. работ #437103

**Comment by audetv** [comment]
Исправлен основной - обычный поисковый индекс. Добавлены стоп слова.
Последний комментарий проиндексированный ранее, и полученный из БД. 
https://svodd.ru/?search%5Bquery%5D=437103&search%5Bmatching%5D=&search%5Bmatching%5D=in&search%5Bdictionary%5D=0

в 00:38 включено обновление БД. После #437103 новые комментарии, переданы парсером.
Работы по исправлению концептуального индекса перенесены на 9:00 МСК

**Reply by iprst** [reply]
Да, обычный поиск проиндексировался целиком и отдаёт комментарии вплоть до [#437106](https://svodd.ru/?search%5Bquery%5D=437106&search%5Bmatching%5D=&search%5Bmatching%5D=in&search%5Bdictionary%5D=0) с правильной датой (сейчас это самый новый комментарий на сайте).

**Comment by audetv** [comment]
28.04.2023 09:10 МСК остановлено обновление БД, производятся работы по обновлению поискового индекса.
Последний комментарий в БД до начала тех. работ #437115
https://svodd.ru/?search%5Bquery%5D=437115&search%5Bmatching%5D=&search%5Bmatching%5D=in&search%5Bdictionary%5D=0&search%5Bdictionary%5D=1
Обновляется индекс концептуального поиска. Примерные сроки завершения работ 10:00 МСК

**Comment by audetv** [comment]
Примерные сроки завершения работ 11:00 МСК

**Comment by audetv** [comment]
28.04.2023 10:39 МСК запуск сервера в обычном режиме, обновления завершено.

**Reply by iprst** [reply]
Работает. Концептуальный и обычный индекс отдают правильные даты.
